lib/helper/Puppeteer.js

```diff
@@ -1418,22 +1418,29 @@ class Puppeteer extends Helper {
     return this.page.evaluate.apply(this.page, args);
   }
 
-
   /**
-   * {{> grabTextFrom }}
+   * {{> grabTextFromAll }}
    * {{ react }}
    */
-  async grabTextFrom(locator) {
+  async grabTextFromAll(locator) {
     const els = await this._locate(locator);
     assertElementExists(els, locator);
     const texts = [];
     for (const el of els) {
       texts.push(await (await el.getProperty('innerText')).jsonValue());
     }
-    if (texts.length === 1) return texts[0];
     return texts;
   }
 
+  /**
+   * {{> grabTextFrom }}
+   * {{ react }}
+   */
+  async grabTextFrom(locator) {
+    const texts = await this.grabTextFromAll(locator);
```

Comment:
Cool, we can improve the experience by adding this line:

```
if (texts.length > 1) this.debug('Using first element out of ' + texts.length);
``` 

There is a similar message accross helper somewhere. I think it will be useful here too

Reply:
Got it, will do it.

Reply:
Made the changes